packages/kbn-build/src/config.ts

```diff
@@ -3,7 +3,10 @@ import { resolve } from 'path';
 /**
  * Returns all the paths where plugins are located
  */
-export function getProjectPaths(rootPath, options) {
+export function getProjectPaths(
+  rootPath: string,
+  options: { [key: string]: any }
```

Comment:
Would be nice to type the options, maybe something along the lines of

```
type ProjectPathOptions = {
  'skip-kibana-extra': boolean,
  'skip-kibana': boolean,
  [key: string]: any
}
```

(haven't tried to make that change, so don't know if it makes sense across the code base)

Reply:
Yeah, I was thinking about that too, but maybe we should omit `[key: string]: any`part and be stricter?

Reply:
++ that would be great

Reply:
:heavy_check_mark: